pkgs/tools/cd-dvd/cdimgtools/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchgit, autoreconfHook, makeWrapper, libdvdcss, libdvdread, perl, perlPackages, asciidoc, xmlto, sourceHighlight, docbook_xsl, docbook_xml_dtd_45 }:
+stdenv.mkDerivation rec {
+  name = "cdimgtools-${version}";
+  version = "0.3";
+
+  src = fetchgit {
+    url = "git://repo.or.cz/cdimgtools.git";
+    rev = "19736f49646d4bcabcc063cc65f6fcaa655eb98c";
+    sha256 = "1hxg3glxj26fh9xnv184sfdc2wi4cnzvn7chz47wcqdsd0d5fn8w";
+  };
+
+  nativeBuildInputs = [ autoreconfHook makeWrapper asciidoc perlPackages.PodPerldoc xmlto sourceHighlight docbook_xsl docbook_xml_dtd_45 ];
+
+  buildInputs = [ perl perlPackages.StringEscape perlPackages.DataHexdumper libdvdcss libdvdread ];
+
+  patches = [ ./nrgtool_fix_my.patch ./removed_dvdcss_interface_2.patch];
+
+  postFixup = ''
+    for cmd in raw96cdconv nrgtool; do
+      wrapProgram "$out/bin/$cmd" --prefix PERL5LIB : "$PERL5LIB"
+    done
+  '';
+
+  outputs = [ "out" "doc" ];
+
+  installTargets = [ "install" "install-doc" ];
+
+  meta = with stdenv.lib; {
+    homepage = https://repo.or.cz/cdimgtools.git;
```

Comment:
Perhaps `https://repo.or.cz/cdimgtools.git/blob_plain/refs/heads/release:/README.html` is better?

Reply:
Done :-)